src/d3a/models/area/markets.py

```diff
@@ -54,20 +56,19 @@ def rotate_markets(self, current_time, stats, dispatcher):
                                   past_markets=self.past_balancing_markets,
                                   area_agent=dispatcher.balancing_agents)
         stats.update_accumulated()
+        self.indexed_future_markets = {
```

Comment:
How about adding this as a property  as it is used twice in this class?

Reply:
I'll do it, thanks!